frontend/src/api/moment/createMoment.ts

```diff
@@ -0,0 +1,73 @@
+import { API, graphqlOperation, Storage } from 'aws-amplify'
+import { ContentType, CreateMomentInput } from '../../API'
+import { extension } from 'mime-types'
+import awsExports from '../../aws-exports'
+import { createMoment } from '../../graphql/mutations'
+
+export interface Media {
+  name: string
+  data: File | string | any
+  type: string
```

Comment:
Why not use `type: ContentType` here? Allows us to get rid of some code below and makes more sense as we have full control of all the types of media that we allow.

```suggestion
  type: ContentType
```

Reply:
Nope thats the mimetype which we don't know